### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/web/api/location/password/index.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ tags:
   - Reference
 browser-compat: api.Location.password
 ---
-{{APIRef("HTMLDOM")}}
+{{ApiRef("Location")}}
```
Comment (nschonni) [inline files/en-us/web/api/location/password/index.md:11]: Note that this doesn't appear to change the rendering, but matches what the other pages in this area use. "HTMLDOM" does not appear anywhere else for APIRef

Comment (teoli2003) [reply files/en-us/web/api/location/password/index.md:11]: This should be:
```suggestion
{{APIRef("HTML DOM")}}
```

File: files/en-us/web/api/location/username/index.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ tags:
   - Reference
 browser-compat: api.Location.username
 ---
-{{APIRef("HTMLDOM")}}
+{{ApiRef("Location")}}
```
Comment (teoli2003) [inline files/en-us/web/api/location/username/index.md:11]:
```suggestion
{{APIRef("HTML DOM")}}
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
Thanks!

**Pull request merged**